docs/docs/code_submission.md

```diff
@@ -61,29 +59,24 @@ robot. It will be used as an overlay on the robots in the simulator.
 The logo needs to be stored in the [PNG format](https://en.wikipedia.org/wiki/Portable_Network_Graphics)
 and located in a file called `logo.png`.
 
-If you choose to add one, the folder structure will looks something like the
+If you choose to add one, the folder structure will look something like the
 following:
 
 ```
 controllers/
-├── robot1/
+├── robot/
+│   └── robot.py
 │   └── robot1.py
-│   └── utils.py
-├── robot2/
 │   └── robot2.py
-├── robot3/
 │   └── robot3.py
+|   └── utils.py
 ├── logo.png
 └── team_name.txt
 ```
 
-
-If you'd like to 
-
-
 ### Create a ZIP archive
 
-This step is very simple: you just pick the **three directories** that contain
+This step is very simple: you just pick **one directory** that contain
```

Comment:
Probably better to say: ... you just pick **THE directory** that containS

Reply:
Fixed.